common/src/Microsoft.Azure.IIoT.Agent.Framework/src/Agent/Default/Worker.cs

```diff
@@ -218,7 +221,7 @@ public class Worker : IWorker, IDisposable {
                     ct.ThrowIfCancellationRequested();
                     using (_jobProcess = new JobProcess(this, jobProcessInstruction,
                         _lifetimeScope, _logger)) {
-                        await _jobProcess.WaitAsync(); // Does not throw
+                        await _jobProcess.WaitAsync(ct); // Does not throw
```

Comment:
When adding new or changing awaitable calls, we could always insert ConfigureAwait(false). At least until we get a global solution :)

Reply:
thanks!